Shared-Layer/ApiServices/AuthenticationService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using Blazored.LocalStorage;
+using Shared_Layer.DTO_s.User;
+
+namespace Shared_Layer.ApiServices
+{
+    public class AuthenticationService : IAuthenticationService
+    {
+        private readonly HttpClient _httpClient;
+        private readonly ILocalStorageService _localStorage;
+
+        public AuthenticationService(HttpClient httpClient, ILocalStorageService localStorage)
+        {
+            _httpClient = httpClient;
+            _localStorage = localStorage;
+        }
+
+        public async Task LoginAsync(LoginUserDTO loginUser)
```

Comment:
It is recomended to use public async Task<bool> , this way the method returns a boolean (true/false), making it easier to handle the success/failure state in the calling code.